Chore: update README (#119)
rejot-dev/rejot: README.md

      <img height="100px" alt="The ReJot logo" src="./resources/rejot-logo-text-black.png">
    </picture>
  </a>
</h1>

<p align="center">
  <b>Supercharged Replication:</b>
  <i>Turn the write-ahead log of your database into an asynchronous communication channel for your services 🚀</i>
</p>

<h4 align="center">
  <a href="https://rejot.dev/docs/start/quickstart/">Quickstart</a>
  |
  <a href="https://rejot.dev/docs/">Docs</a>
  |
  <a href="https://x.com/ReJotSync">Follow on X</a>
</h4>

<p align="center">
    <img src="./resources/rejot-system-overview.webp" alt="ReJot's data catalog system overview" width="90%">
</p>

## Introduction

ReJot is the missing middle-ground solution between synchronous inter-service communication
(REST/SOAP) and asynchronous event streaming (like Kafka).

Our solution is based on database replication with a focus on developers (as opposed to
infrastructure). With ReJot, you define what to publish and consume using your database's query
language. The transformations and schemas are co-located with a service's internal data model.

We solve the problem of fragile service calls and compounding latency issues that arise from

> synchronous micro-service communication by being an asynchronous alternative. On the other hand,

Reviewer: ```suggestion
synchronous microservice communication by being an asynchronous alternative. At the same time,
```

Reviewer: Maybe:

At the same time we can avoid the operational complexity that Kafka brings by re-using your database's changelog as a queue. Making ReJot a lightweight addition to your infrastructure.
